src/main/webapp/WEB-INF/views/admin/configuration/edit-js.jsp

```diff
@@ -139,11 +139,10 @@
 			var userName = $('#userInfoArea input[name="userName"]').val();
 			var division = $.trim($('#userInfoArea select[name="division"] option:selected').val());
 			var confirmMsg = "";
-			if(division === ""){
-			    division = "${ct:getConstDef('CD_USER_DIVISION_EMPTY')}";
+			if(division === "${ct:getConstDef('CD_USER_DIVISION_EMPTY')}"){
```

Comment:
What if division is null?

Reply:
Because division value is read from the selected item, there is no case when division is null.